modules/blocks.php

```diff
@@ -80,3 +87,62 @@ function jetpack_map_block_load_assets( $attr, $content ) {
 	Jetpack_Gutenberg::load_assets_as_required( 'map', $dependencies );
 	return preg_replace( '/<div /', '<div data-api-key="'. esc_attr( $api_key ) .'" ', $content, 1 );
 }
+
+/**
+ * Mailchimp block registration/dependency declaration.
+ *
+ * @param array $attr - Array containing the map block attributes.
+ *
+ * @return string
+ */
+function jetpack_mailchimp_block_load_assets( $attr ) {
```

Comment:
There is no way we could repurpose the shortcode code for this? This seems like a lot of code duplication

Reply:
I don't think we will need the shortcode at all once this makes it in. This way the block will exist and be useable even if you don't have the Shortcodes module on.

Reply:
> I don't think we will need the shortcode at all once this makes it in

I disagree, but my perspective may be totally wrong :)
I started a discussion here p58i-7ER-p2

> This way the block will exist and be useable even if you don't have the Shortcodes module on.

Good point.